CMakeLists.txt

```diff
@@ -778,6 +778,12 @@ add_subdirectory(test)
 if (CMAKE_BUILD_TYPE STREQUAL "Debug")
     message(AUTHOR_WARNING "Benchmarks are available only in release build")
 else ()
+    if(ENABLE_BUNDLED_LIBBENCHMARK)
+        include(BuildBenchmark)
+        add_dependencies(build_bundled_libs bundled-libbenchmark)
+    else()
+        set(BENCHMARK_LIBRARIES benchmark::benchmark)
+    endif()
```

Comment:
I propose to move this hunk to `perf/CMakeLists.txt`

Reply:
Moved as you suggested.
```diff
diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1fc7258adc..a8a3990317 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -778,12 +778,6 @@ add_subdirectory(test)
 if (CMAKE_BUILD_TYPE STREQUAL "Debug")
     message(AUTHOR_WARNING "Benchmarks are available only in release build")
 else ()
-    if(ENABLE_BUNDLED_LIBBENCHMARK)
-        include(BuildBenchmark)
-        add_dependencies(build_bundled_libs bundled-libbenchmark)
-    else()
-        set(BENCHMARK_LIBRARIES benchmark::benchmark)
-    endif()
     add_subdirectory(perf)
 endif ()
 add_subdirectory(doc)
diff --git a/perf/CMakeLists.txt b/perf/CMakeLists.txt
index 12fda73011..82f3a8df62 100644
--- a/perf/CMakeLists.txt
+++ b/perf/CMakeLists.txt
@@ -30,7 +30,10 @@ if(NOT ENABLE_BUNDLED_LIBBENCHMARK)
         )
         return()
     endif()
+    set(BENCHMARK_LIBRARIES benchmark::benchmark)
 else()
+    include(BuildBenchmark)
+    add_dependencies(build_bundled_libs bundled-libbenchmark)
     include_directories(${BENCHMARK_INCLUDE_DIRS})
 endif()
 
```

Reply:
Thanks!